### PR TITLE
on 403, more info

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

**Comment by volodymyrss** [comment]
test to be adapted once dispacher-app interface merges

File: cdci_osa_plugin/osa_dataserve_dispatcher.py [hunk 1]
```diff
@@ -688,15 +688,19 @@ def check_query_roles(self, provided_roles: List[str], par_dic: dict):
         # integral_data_rights = self.get_par_by_name('integral_data_rights').value
 
         needed_roles = []
+        needed_roles_with_comments = {}
```
Comment (burnout87) [inline cdci_osa_plugin/osa_dataserve_dispatcher.py:691]: I guess this will have to be returned, along with needed_roles

**Review by burnout87 — CHANGES_REQUESTED** [review]

**Comment by volodymyrss** [comment]
until no test it's draft

**Comment by volodymyrss** [comment]
combined with #37 which is similar

**Pull request closed without merging**